src/main/resources/log4j.properties

```diff
@@ -15,3 +15,13 @@ log4j.appender.CONSOLE.layout=org.apache.log4j.PatternLayout
 # for details on the pattern layout string.
 log4j.appender.CONSOLE.layout.conversionPattern=%d{dd MMM yyyy HH:mm:ss,SSS} [%-5p] [%C{1}]: %m%n
 
+log4j.appender.STDERR=org.apache.log4j.ConsoleAppender
+log4j.appender.STDERR.layout=org.apache.log4j.PatternLayout
+log4j.appender.STDERR.layout.conversionPattern=%d{dd MMM yyyy HH:mm:ss,SSS} [%-5p] [%C{1}]: %m%n
+log4j.appender.STDERR.Threshold=WARN
+log4j.appender.STDERR.Target=System.err
+
+
+#log4j.logger.software.amazon.awssdk=DEBUG, CONSOLE
```

Comment:
Are these comments needed?

Reply:
yup yup, for the next peasant that has to debug it, they can uncomment